src/RSSFeed.php

```diff
@@ -1118,6 +1119,7 @@ public function getRights($interface = 'central')
             $values = [READ => __('Read')];
         } else {
             $values = parent::getRights();
+            $values[self::PERSONAL] = __('Show personal when no public rights');
```

Comment:
Right should not be limited to `show` feature. If user has no right to show its personal feeds, he should also not be able to create, update, delete and purge its own feeds.
In this case, it probably means that, when a feed is created, its visibility should be forced to "public".